src/Link.tsx

```diff
@@ -1,14 +1,15 @@
 import * as React from 'react';
-import { Box, IBox } from './Box';
+import { Box, IBox, IBoxCSS } from './Box';
 import { useTheme } from './theme';
 
 export interface ILink extends IBox<HTMLAnchorElement> {}
 
 export const Link = React.forwardRef<HTMLAnchorElement, ILink>((props, ref) => {
-  return <Box {...props} as="a" ref={ref} css={linkStyles()} />;
+  const { css, ...rest } = props;
+  return <Box {...rest} as="a" ref={ref} css={linkStyles(css)} />;
```

Comment:
Minor nitpick, I noticed in most places you use css as an array, and in a few you pass css into the style function for the component, like here. Can you make this an array since thats the majority convention and css doesn't need to get used in the linkStyles function

Reply:
The `linkStyles` function actually returns an array already. The reason it's kind of mixed up is because I did the minimal amount of changes to the existing `xyzStyles` functions in the components, and some of them return objects while some return arrays. Does that make sense?